File: src/client/java/com/somefrills/events/HudRenderEvent.java
package com.somefrills.events;

import net.minecraft.client.DeltaTracker;
import net.minecraft.client.gui.Font;
import net.minecraft.client.gui.GuiGraphics;

public class HudRenderEvent {
    public GuiGraphics context;
    public Font textRenderer;
    public DeltaTracker tickCounter;

    public HudRenderEvent(GuiGraphics context, Font textRenderer, DeltaTracker tickCounter) {
        this.context = context;
        this.textRenderer = textRenderer;
        this.tickCounter = tickCounter;
    }
}

File: src/main/kotlin/com/somefrills/events/HudRenderEvent.kt
package com.somefrills.events

import net.minecraft.client.DeltaTracker
import net.minecraft.client.gui.Font
import net.minecraft.client.gui.GuiGraphicsExtractor

class HudRenderEvent(val context: GuiGraphicsExtractor, val textRenderer: Font, val tickCounter: DeltaTracker) :
    FrillsEvent()
